tests/modules/nf-core/somalier/ancestry/nextflow.config

```diff
@@ -0,0 +1,15 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: SOMALIER_EXTRACT {
+        cpus = 1
```

Comment:
Why are you specifying the `cpus` here? They should default to 2 for the `nf-core/modules` repo